db/change_cache.go

```diff
@@ -90,6 +89,10 @@ func (c *changeCache) updateStats(ctx context.Context) {
 	c.db.DbStats.Cache().PendingSeqLen.Set(int64(c.internalStats.pendingSeqLen))
 	c.db.DbStats.CBLReplicationPull().MaxPending.SetIfMax(int64(c.internalStats.maxPending))
 	c.db.DbStats.Cache().HighSeqStable.Set(int64(c._getMaxStableCached(ctx)))
+	c.db.DbStats.Cache().SkippedSeqLen.Set(c.skippedSeqs.getSliceLength())
```

Comment:
Is there something non-standard going on with the whitespace here?  (spaces instead of tab, maybe?)

Reply:
I don't think so? I have updated to reflect the changes to have a getStats() call for the skipped slice.